lib/taste_tester/commands.rb

```diff
@@ -99,7 +101,7 @@ def self.test
           tested_hosts << hostname
         end
       end
-      unless TasteTester::Config.skip_post_test_hook
+      unless TasteTester::Config.skip_post_test_hook || TasteTester::Config.linkonly
```

Comment:
I'm not convinced always skipping the hooks with -L makes sense... but I don't feel strongly.. but we don't know what people may put in their plugins.